src/process_utility.c

```diff
@@ -2588,7 +2588,7 @@ process_create_trigger_start(ProcessUtilityArgs *args)
 	CreateTrigStmt *stmt = (CreateTrigStmt *) args->parsetree;
 	Cache *hcache;
 	Hypertable *ht;
-	ObjectAddress address;
+	ObjectAddress __attribute__((unused)) address;
```

Comment:
```suggestion
	ObjectAddress pg_attribute_unused() address;
```
`__attribute__((unused))` is not well supported

Reply:
```suggestion
	ObjectAddress PG_USED_FOR_ASSERTS_ONLY address;
```
actually